pom.xml

```diff
@@ -206,7 +180,6 @@
                         </goals>
                         <configuration>
                             <projectsDirectory>src/test/it</projectsDirectory>
-                            <writeJunitReport>true</writeJunitReport>
```

Comment:
doesn't seem to exist as a property, I couldn't find any info on it

Reply:
Required 3.1.2, https://github.com/jenkinsci/view-job-filters-plugin/pull/29/commits/09569c2bf7ecec4acb72da7e486dd2787ae67713